lib/matplotlib/tests/test_transforms.py

```diff
@@ -621,28 +621,6 @@ def test_invalid_arguments():
         t.transform([[1, 2, 3]])
 
 
-def test_transformed_path():
```

Comment:
This test is incorrect, so I've removed it. Line 640 was a no-op, because `path` never uses an attribute called `points`. On the other hand, changing `.points` to `._vertices` fails the test, because the cached transformed path is the same as `path` as `trans` is a purely affine transform.

Reply:
The other part is still a valid check, is it not?

Reply:
I guess it is. I'll restore it.